### ISSUE TITLE
dcsr.bit18 is confilct with other rv spec

### ISSUE THREAD

**Issue by jiajiahuhuRa** [riscv-non-isa/riscv-external-debug-security#74]
hi,SPEC RISC-V Shadow Stacks and Landing Pads（Version v1.0, 2024-07-03: Ratified) introduces the pelp (bit 18) in dcsr and is conflict with  the current spec

**Comment by pdonahue-ventana** [comment]
I agree.  See https://github.com/riscv/riscv-debug-spec/blob/main/xml/core_registers.xml#L118.

**Comment by AoteJin** [comment]
Thanks for speaking out. I also noticed it and moved it to bit 20 in commit https://github.com/riscv-non-isa/riscv-external-debug-security/commit/ca71caf0fe4a140f59369e73562146f795532f6f. Will close the if you are fine with the change.
